textblob/blob.py

```diff
@@ -148,6 +148,16 @@ def lemmatize(self, pos=None):
         lemmatizer = nltk.stem.WordNetLemmatizer()
         return lemmatizer.lemmatize(self.string, pos)
 
+    #added 'stemmer' on lines of lemmatizer
+    #based on nltk
+    def stem(self, isporter=True):
```

Comment:
Are there other stemmers available in NLTK? If so, I'm not sure a boolean param will suffice here. Perhaps the user could pass an instance of a stemmer, e.g.

```python
blob.stem(stemmer=PorterStemmer())
```